### PR TITLE
Kokawata/opa

### PR DESCRIPTION
**Pull request by koheikawata**
- Add the OPA policy to example.rego
- Add the OPA policy test to example_test.rego

### CODE REVIEW

File: src/opa/rego/policy.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "authorization_rules": [    
+    {"route": "/users", "roles": ["admin"]},
+    {"route": "/order", "roles": ["contributor"]},
+    {"route": "/users/*/friends", "roles": ["admin", "contributor"]}
```
Comment (oottka) [inline src/opa/rego/policy.json]: To get this working end-to-end we'll need to add some of the routes in the conference API. That can be in another PR as well thought

File: src/opa/rego/example.rego [hunk 2]
```diff
@@ -2,10 +2,12 @@ package rbac
...
-# cares about the payload, so we ignore the others.
-token = {"header": header, "payload": payload, "signature": signature} { io.jwt.decode(input.token, [header, payload, signature]) }
+token = {"header": header, "payload": payload, "signature": signature} { io.jwt.decode(input.input.token, [header, payload, signature]) }
 
-# By default, we deny the request
 default allow = false
+
+allow {
+  some i
+  glob.match(data.authorization_rules[i].route, [], input.input.route)
```
Comment (oottka) [inline src/opa/rego/example.rego]: I wonder if input twice is correct - @MahrRah ?

**Review by oottka — APPROVED** [review]

File: src/opa/rego/policy.json (hunk 1 above)
Comment (koheikawata) [reply src/opa/rego/policy.json]: I changed routes in policy.json using conference API's routes

File: src/opa/rego/example.rego (hunk 2 above)
Comment (koheikawata) [reply src/opa/rego/example.rego]: I also fixed json file and remove one `input`
